From d7d034322eaabe86f3693609089d15c23a4f6cbc Mon Sep 17 00:00:00 2001
From: Arik Nemtsov <arik@wizery.com>
Date: Thu, 17 Mar 2011 12:22:48 -0500
Subject: [PATCH] wpa_supplicant: include libnl in build dependencies

This is required for using the libnl functionality in the custom wext
driver used for the wl12xx solution.

Change-Id: I0fb0c6bdaeff4df0bbd1be5c8aaf1271bb05be2f
---
 wpa_supplicant/Android.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/wpa_supplicant/Android.mk b/wpa_supplicant/Android.mk
index d3ab3dd..cd6b5f2 100644
--- a/wpa_supplicant/Android.mk
+++ b/wpa_supplicant/Android.mk
@@ -1127,7 +1127,7 @@ endif
 ifneq ($(BOARD_WPA_SUPPLICANT_PRIVATE_LIB),)
 LOCAL_STATIC_LIBRARIES += $(BOARD_WPA_SUPPLICANT_PRIVATE_LIB)
 endif
-LOCAL_SHARED_LIBRARIES := libc libcutils libcrypto libssl
+LOCAL_SHARED_LIBRARIES := libc libcutils libcrypto libssl libnl
 LOCAL_CFLAGS := $(L_CFLAGS)
 LOCAL_SRC_FILES := $(OBJS)
 LOCAL_C_INCLUDES := $(INCLUDES)
-- 
1.7.1

